README.md

```diff
@@ -0,0 +1,62 @@
+# Optimove-SDK-Unity
```

Comment:
Worth a note or table about supported unity versions?

Reply:
so far we support >= 2019.3 and only have 1 sdk version -- added a note